docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -2905,7 +2905,28 @@ empty(array)
 
 ### `generate_series`
 
-_Alias of [range](#range)._
+similar as range function, but include the upper bound
```

Comment:
```suggestion
Similar to the range function, but it includes the upper bound. 
```

Reply:
Sorry I rebased the branch and force pushed, I will add your document change manually